src/main/java/com/faforever/client/rankedmatch/Ladder1v1Controller.java

```diff
@@ -138,7 +138,7 @@ public void initialize() {
     for (Faction faction : EnumSet.of(AEON, CYBRAN, UEF, SERAPHIM)) {
       factionsToButtons.get(faction).setSelected(factions.contains(faction));
     }
-    playButton.setDisable(factionsToButtons.values().stream().anyMatch(ToggleButton::isSelected));
+    playButton.setDisable(!factionsToButtons.values().stream().anyMatch(ToggleButton::isSelected));
```

Comment:
As IntelliJ suggests, `!anyMatch()` should be replaced by `noneMatch()` :)
![image](https://user-images.githubusercontent.com/1864183/35766451-5f925d8a-08d9-11e8-8722-2bae3a57b01a.png)